### DISCUSSION TITLE
How to ignore my own visits when the IP is dynamic?

### DISCUSSION THREAD

**Discussion by vitobotta** [umami-software/umami#2305] [Q&A] [answered]
Hi! So far I have been using the environment variable to ignore my own visits, but my IP is now dynamic (I am forced to use a VPN). Is there a way to exclude my visits from the tracking even with a dynamic IP?

Thanks!

**Comment by briancao** [comment, marked as answer]
Add the key `umami.disabled` and set its value to true in localStorage.

**Reply by vitobotta** [reply]
Thanks!

**Reply by jsjoshua** [reply]
Is there a way to exclude visits from my cell phone, which isn't local and has a dynamic IP address when not on my home network?  I suppose I could use a VPN to visit my site, but is there any systemic way to do this?

**Reply by supermario0711** [reply]
+1 on mobile question. Any chance you can help?

**Reply by jsjoshua** [reply]
I never solved this. Sorry.
